patterns/footer-social.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: Centered Footer with social links
+ * Slug: twentytwentyfive/footer-social
+ * Categories: footer
+ * Block Types: core/template-part/footer
+ * Description: Footer with centered site title and social links
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
```

Comment:
I have not changed the background and text color on this footer because I think those colors should be applied by a section style, which has not been created yet.